packages/build_cmd/README.md

```diff
@@ -0,0 +1,2 @@
+A sample command-line application with an entrypoint in `bin/`, library code
```

Comment:
Is it a sample or a tool? Should we have a tool directory for components like this?

Reply:
Ha, this generated code really sneaks by ya. Updated!